contrib/scripts/push_manifest.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+IMAGE_NAME=${1:-}
+IMAGE_TAG=${2:-latest}
+DOCKER_REPOSITORY=${3:-cilium}
+DOCKER_REGISTRY=${4:-}
+IMAGE_ARCH=("amd64" "arm64")
+
+function using_help() {
+  echo "Please specify a image name!"
+  echo -e "\nUsage::\n\tpush_manifest.sh IMAGE_NAME [IMAGE_TAG] [DOCKER_REPOSITORY] [DOCKER_REGISTRY]"
+  echo -e "\nExample::\n\tpush_manifest.sh cilium-runtime latest"
+  exit 1
+}
+
+function install_manifest() {
+  MANIFEST_VERSION=v1.0.0
+  rm -f ./manifest-tool
+  wget https://github.com/estesp/manifest-tool/releases/download/${MANIFEST_VERSION}/manifest-tool-linux-${ARCH} \
```

Comment:
Do we need this tool? Are we using any functionality not available in `docker manifest`?

Reply:
In this PR, initially I used docker manifest to implement multi-arch. Later, considered that this would have a dependency on a specific docker version, so separate manifest-tool  is used for manifest operations.

Reply:
I have learnt the other day that [buildx is offers the way of building multi-arch images](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images), perhaps something to look into?

Reply:
Yep buildx works pretty well! We use it for https://github.com/scaleway/scaleway-cloud-controller-manager and so far so good

Reply:
Docker Buildx is included in Docker 19.03, depending on a particular version,
I really don't recommend it.

Reply:
Agree, it also depends on the OS..